app/Console/Command/ConvertClusterIdsToHashes.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Doctrine\DBAL\Connection;
+use Symfony\Component\Console\Command\Command as BaseCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/*
+ * @todo This is a temporary script and can probably be removed somewhere in 2025
```

Comment:
I would make a ticket for this; this can be removed the first time this has run.